aggregate_spatial_window.json

```diff
@@ -0,0 +1,96 @@
+{
+    "id": "aggregate_spatial_window",
+    "summary": "Zonal statistics for rectangular windows",
+    "description": "Aggregates statistics over the spatial dimensions. This process passes a list of values to the reducer. \n\n The results are computed dividing the input X/Y grid into xDim/yDim non-overlapping windows and applying the reducer process to each window.\n If X and Y are not multiple of xDim/yDim NaNs padding before the computation is applied.",
```

Comment:
You need to remove all spaces around \n, otherwise the linter fails.

Reply:
Spaces removed.